src/tools/interop/idt/capture/ecosystem/play_services/play_services_analysis.py

```diff
@@ -37,10 +42,16 @@ def __init__(self, platform: Android, artifact_dir: str) -> None:
         self.resolver_logs = ''
         self.sigma_logs = ''
         self.fail_trace_line_counter = -1
+        self.real_time = True
+
+    def rt_log(self, line):
+        if self.real_time:
+            self.logger.info(line)
 
     def _log_proc_matter_commissioner(self, line: str) -> None:
         """Core commissioning flow"""
         if 'MatterCommissioner' in line:
+            self.rt_log(line)
```

Comment:
This rt_log function isn't needed after the move away from multiproc; will remove.